phpcs.xml

```diff
@@ -4,6 +4,12 @@
 
     <rule ref="CakePHP"/>
 
+    <rule ref="SlevomatCodingStandard.Namespaces.UnusedUses">
```

Comment:
The tests do not contain all necessary imports (uses) right now because without this PHPCS will complain.
I added the missing imports to MigrateTest and will leave the rest for someone else to clean up.